plugins/field-colour/README.md

```diff
@@ -30,11 +32,15 @@ This field accepts up to 4 parameters:
 - "columns" to specify the number of columns the colour dropdown should have.
   Defaults to 7.
 
-### JavaScript
+If you want to use only the field, you must install it by calling `registerFieldColour` before instantiating your blocks. If another field is registered under the same name, this field will overwrite it.
```

Comment:
Wording: they must register it, but calling that function is not the only possible way to do so as the wording here implies.